sdk/communication/communication-verified-recipients/README.md

```diff
@@ -0,0 +1,192 @@
+Verification README
+
+# Azure Communication Verified Recipients client library for JavaScript
+
+The Verified Recipient library allows users to verify numbers for send messages or calling.
+
+## Getting started
+
+### Prerequisites
+
+- An [Azure subscription][azure_sub].
+- An existing Communication Services resource. If you need to create the resource, you can use the [Azure Portal][azure_portal], the [Azure PowerShell][azure_powershell], or the [Azure CLI][azure_cli].
+
+### Installing
+
+```bash
+npm install @azure-tools/communication-verified-recipients
```

Comment:
wait.. is it meant to be put under `@azure-tools/` scope?

Reply:
yea that is intentional